core/domain/services/graphql/types/Ticket.php

```diff
@@ -67,82 +67,46 @@ public function getFields()
                 'ID',
                 esc_html__('Ticket ID', 'event_espresso')
             ),
-            new GraphQLField(
-                'name',
-                'String',
-                'name',
-                esc_html__('Ticket Name', 'event_espresso')
+            new GraphQLInputField(
+                'datetimes',
+                ['list_of' => 'ID'],
+                null,
+                sprintf(
+                    '%1$s %2$s',
+                    esc_html__('Globally unique IDs of the datetimes related to the ticket.', 'event_espresso'),
+                    esc_html__('Ignored if empty.', 'event_espresso')
+                )
```

Comment:
why separate strings?
why not just:

```php
esc_html__('Globally unique IDs of the datetimes related to the ticket. Ignored if empty.', 'event_espresso')
```

Reply:
That makes the translation of `Ignored if empty.` reusable somewhere else. :)